tests/e2e/workloads/bdi/conftest.py

```diff
@@ -0,0 +1,452 @@
+
+
+@pytest.fixture(scope="class")
+def create_secretes_factory_class(request):
+    return create_secretes_factory_fixture(request)
+
+
+@pytest.fixture(scope="session")
+def create_secretes_factory_session(request):
+    return create_secretes_factory_fixture(request)
+
+
+@pytest.fixture(scope="function")
+def create_secretes_factory(request):
+    return create_secretes_factory_fixture(request)
+
+
+def create_secretes_factory_fixture(request):
+    """
+    Create DB2U & LDAP secrets
+    """
+    temp_ldap_r_n = []
+    temp_db2u_r_n = []
+    ocp_proj = []
+    db2u_project = []
+
+    def factory(db2u_project_name, ldap_r_n, ldap_r_p, db2u_r_n, db2u_r_p):
+        """
+        Args:
+            db2u_project_name (str): Name of the db2u project.
+            ldap_r_n (str): LDAP release name.
+            ldap_r_p (str): LDAP release password.
+            db2u_r_n (str): DB2U release name
+            db2u_r_p (str): DB2U release name
+        """
+        db2u_project.append(db2u_project_name)
+        ocp_proj.append(ocp.OCP(namespace=db2u_project_name))
+        temp_ldap_r_n.append(ldap_r_n)
+        temp_db2u_r_n.append(db2u_r_n)
+
+        log.info("Creating LDAP secrets")
+        ocp_proj[0].exec_oc_cmd(
+            command=f"create secret generic {ldap_r_n}-db2u-ldap-bluadmin "
+            f"--from-literal=password={ldap_r_p}"
+        )
+
+        log.info("Creating DB2U secrets")
+        ocp_proj[0].exec_oc_cmd(
+            command=f"create secret generic {db2u_r_n}-db2u-instance "
+            f"--from-literal=password={db2u_r_p}"
+        )
+
+    def finalizer():
+        """
+        Delete secrets
+        """
+        ocp_proj[0].exec_oc_cmd(
+            command=f"delete -n {db2u_project[0]} "
+            f"secret/{temp_ldap_r_n[0]}-db2u-ldap-bluadmin "
+            f"secret/{temp_db2u_r_n[0]}-db2u-instance"
+        )
+
+    request.addfinalizer(finalizer)
+    return factory
+
+
+@pytest.fixture(scope="class")
+def create_ibm_container_registry_factory_class(request):
+    return create_ibm_container_registry_factory_fixture(request)
+
+
+@pytest.fixture(scope="session")
+def create_ibm_container_registry_factory_session(request):
+    return create_ibm_container_registry_factory_fixture(request)
+
+
+@pytest.fixture(scope="function")
+def create_ibm_container_registry_factory(request):
+    return create_ibm_container_registry_factory_fixture(request)
+
+
+def create_ibm_container_registry_factory_fixture(request):
+    """
+    Register with IBM could in order to get the DB2 Warehouse container images
+
+    """
+
+    def factory(db2u_project_name, ibm_cloud_key):
+        """
+        Args:
+            db2u_project_name (str): Name of the db2u project.
+            ibm_cloud_key (str): IBM cloud key for image pulling
+
+        """
+
+        ocp_proj = ocp.OCP(namespace=db2u_project_name)
+
+        log.info("Registering with IBM could for container images")
+        ocp_proj.exec_oc_cmd(
```

Comment:
You should use secrets in exec_oc_cmd to prevent to print secrets data to console output.

see:
https://github.com/red-hat-storage/ocs-ci/blob/41d52f8a9b2e6cbffa923ace5b0e8c02417870a8/ocs_ci/ocs/ocp.py#L114

Reply:
Correct. Done